classes/module/Module.php

```diff
@@ -1045,6 +1060,125 @@ public function editExceptions($id_hook, $excepts)
         return $result;
     }
 
+    /**
+     * Install a tab according to its defined structure
+     *
+     * @param array $tab_info The structure of the tab.
+     *
+     * @return bool   true if the tab was installed successfully, false otherwise
+     *                (and a descriptive error will be added to $_errors)
+     */
+    private function installTab($tab_info)
```

Comment:
I'm pretty sure this function definition will break the shop with many modules.

I usually see the following definition:
```php
	public function installTab()
	{
		// Wololo
	}
```

An error will be thrown if the parameters or the visibility differ. Maybe we should have another function name in order to avoid any compatibility issue. Same thing for `uninstallTab()`